ios/Capacitor/Capacitor/CAPBridgeViewController.swift

```diff
@@ -67,7 +67,7 @@ class CAPBridgeViewController: UIViewController, WKScriptMessageHandler, WKUIDel
     startWebServer(port: port!)
 
     print("⚡️  Loading index.html...")
-    let request = URLRequest(url: URL(string: "http://localhost:\(port!)/")!)
+    let request = URLRequest(url: URL(string: "http://localhost:\(port!)\(indexPath)")!)
```

Comment:
We can't request index.html here, we need the root slash path for proper SPA routing. Many routers rely on `/` to be the index path